Tests/SkipFoundationTests/DateTime/TestCalendar.swift

```diff
@@ -576,6 +576,26 @@ class TestCalendar: XCTestCase {
         XCTAssertEqual(eraRange, 1..<8)
     }
     
+    func testRangeOfComponents() {
+        let calendar = Calendar(identifier: .gregorian)
+        let date = Date(timeIntervalSince1970: 1539146705)
+
+        let monthRangeInAYear = calendar.range(of: .month, in: .year, for: date)
+        XCTAssertEqual(monthRangeInAYear, 1..<13)
+        
+        let dayRangeInAYear = calendar.range(of: .day, in: .year, for: date)
+        XCTAssertEqual(dayRangeInAYear, 1..<366)
+        
+        let weekOfYearRangeInAYear = calendar.range(of: .weekOfYear, in: .year, for: date)
+        XCTAssertEqual(weekOfYearRangeInAYear, 1..<54)
```

Comment:
Curious … this is [failing](https://github.com/skiptools/skip-foundation/pull/28/checks#step:9:28221) on the Java side of the local tests: `java.lang.AssertionError: expected:<1..53> but was:<1..52>`

Reply:
unfortunately I have no idea what has happen to my env. but I don't think my Java tests are working at all.
getting this on every file kinda thing. 
>Annotation argument must be a compile-time constant.
>Unresolved reference 'ext'.

Reply:
That was a recent fix in the skip-unit framework. Can you run `Update to latest package versions` and do a clean build and let me know if the error still happens?

Reply:
yea, got it working. Did some testing and it seems Swift for weeks in a year always returns max which is not an actual max with in given date (year)